.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:jest/recommended',
+    'plugin:prettier/recommended',

```

Comment:
Various recommended rules for ESLint. Here's the [documentation](https://eslint.org/docs/user-guide/configuring#extending-configuration-files) for ESLint extensions.

- `eslint:recommended` Recommended ESLint rules. This comes with the **eslint** module and can be optionally added. Adding it here because they take care of 95% of the basic rules for us. [Documentation here](https://eslint.org/docs/user-guide/configuring#using-eslintrecommended)
- `plugin:react/recommended` Recommended React rules. This comes with the **eslint-plugin-react** module. Without this, ESLint will complain about imported components never being used even though it is. [Documentation here](https://github.com/yannickcr/eslint-plugin-react#configuration)
![image](https://user-images.githubusercontent.com/13035560/60785714-e9cc8c00-a121-11e9-9781-915f37fd2ea3.png)
- `plugin:jest/recommended` Recommended Jest rules. Comes with the corresponding just like React plugin. Without this, ESLint will complain about Jest globals in test files. [Documentation here](https://github.com/jest-community/eslint-plugin-jest#usage)
![image](https://user-images.githubusercontent.com/13035560/60785844-67909780-a122-11e9-9512-5f8c5d05d9bc.png)
- `plugin:prettier/recommended` [Documentation here](https://prettier.io/docs/en/integrating-with-linters.html)